scripts/semver-check.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -Eeuxo pipefail
+set -eo pipefail
```

Comment:
Any reasons to exlude -u and allow undefined variables?

Reply:
This part is copied from gl-native.

Reply:
Is the whole script copied?

Reply:
Yes, I only add some log message

Reply:
I see.
Imagine would be nice to have some common source for such scripts to avoid duplicating code reviews and source of bugs